<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScopeFoundry</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
    <item>
     <widget class="QGroupBox" name="lq_trees_groupBox">
      <property name="title">
       <string>Logged Quantities</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTreeWidget" name="hardware_treeWidget">
          <column>
           <property name="text">
            <string>Hardware</string>
           </property>
          </column>
         </widget>
         <widget class="QTreeWidget" name="measurements_treeWidget">
          <column>
           <property name="text">
            <string>Measurements</string>
           </property>
          </column>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="settings_groupBox">
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="1" column="0">
           <widget class="QPushButton" name="settings_load_last_pushButton">
            <property name="text">
             <string>Load Last</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="settings_autosave_pushButton">
            <property name="text">
             <string>Auto Save</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="settings_save_pushButton">
            <property name="text">
             <string>Save...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="settings_load_pushButton">
            <property name="text">
             <string>Load...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="console_pushButton">
            <property name="text">
             <string>Console...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>438</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
